integration-test/src/test/java/com/datastrato/gravitino/integration/test/catalog/jdbc/postgresql/TestPostgreSqlTableOperations.java

```diff
@@ -152,18 +155,16 @@ public void testOperationTable() {
 
     // alter column Nullability
     TABLE_OPERATIONS.alterTable(
-        TEST_DB_NAME,
-        newName,
-        TableChange.updateColumnNullability(new String[] {"col_1_new"}, false),
-        TableChange.updateColumnNullability(new String[] {"col_2"}, true));
+        TEST_DB_NAME, newName, TableChange.updateColumnNullability(new String[] {"col_2"}, true));
     load = TABLE_OPERATIONS.load(TEST_DB_NAME, newName);
     alterColumns.clear();
     alterColumns.add(
         new JdbcColumn.Builder()
             .withName("col_1_new")
-            .withType(VARCHAR)
+            .withType(Types.LongType.get())
             .withComment("test_new_comment")
             .withNullable(false)
+            .withAutoIncrement(true)
             .build());
     alterColumns.add(
         new JdbcColumn.Builder()
```

Comment:
Can you please add tests verify that

1. auto increment is successfully set in the underlying DB.
2. auto increment can be gotten correctly from the underlying DB.